dev/reference/sql/statements/admin.md

```diff
@@ -8,16 +8,47 @@ category: reference
 
 This statement is a TiDB extension syntax, used to view the status of TiDB and check the data of tables in TiDB.
 
+To view the currently running DDL jobs, use `ADMIN SHOW DDL`:
+
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL;
+```
+
+To view all the results in the current DDL job queue (including tasks that are running and waiting to be run) and the last ten results in the completed DDL job queue, use `ADMIN SHOW DDL JOBS`:
+{{< copyable "sql" >}}
+
+```sql
+ADMIN SHOW DDL JOBS [NUM] [WHERE where_condition];
+```
+
+* `NUM`: To view the last `NUM` results in the completed DDL job queue. If not specified, `NUM` is by default 10.
```

Comment:
Generally, we use a lowercase letter to begin the text immediately following a colon.

```suggestion
* `NUM`: to view the last `NUM` results in the completed DDL job queue. If not specified, `NUM` is by default 10.
```